Bottom: bdef5cd2b170f152c96af349f59d0e97bcb23b63
Top:    bbcc0a1bfa49dd01790dee6bf27898b4dde2733e
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-29 10:18:56 -0500

Refresh of fixup-asset-host

---
diff --git a/roles/buildbot-worker-ptxdist/tasks/main.yml b/roles/buildbot-worker-ptxdist/tasks/main.yml
index f46b0124..9e5bafb7 100644
--- a/roles/buildbot-worker-ptxdist/tasks/main.yml
+++ b/roles/buildbot-worker-ptxdist/tasks/main.yml
@@ -25,7 +25,7 @@
     - '{{ cache_volume }}:{{ cache_path }}:z'
     detach: no
     cleanup: yes
-    command: /bin/ash -c 
+    command: /bin/ash -c
       "mkdir -p {{item}} &&
       chown -c -R {{buildbot_uid}}:{{buildbot_uid}} {{item}} &&
       chmod -v 777 {{item}}"
@@ -44,7 +44,7 @@
     - '{{ data_volume }}:{{ data_path }}:z'
     detach: no
     cleanup: yes
-    command: /bin/ash -c 
+    command: /bin/ash -c
       "mkdir -p {{item}} &&
       if [ $(stat -c '%u' '{{item}}') -ne {{buildbot_uid}} ]; then
         chown -c -R {{buildbot_uid}}:{{buildbot_uid}} {{item}};
@@ -107,6 +107,7 @@
 
 - name: Start buildbot worker container
   docker_container:
+    etc_hosts: '{{ container_etc_hosts }}'
     hostname: '{{ container_hostname }}'
     name: '{{ container_name }}'
     image: '{{ image_name }}:{{ docker_image_tag }}'
